build.gradle.kts

```diff
@@ -455,6 +455,8 @@ project(":here-naksha-lib-handlers") {
 
         implementation(commons_lang3)
         implementation(commons_dbutils)
+
+        setOverallCoverage(0.0)
```

Comment:
Was setting that necessary due to gradle task failure? Just a question, not a remark

Reply:
Yes it was required due to gradle task fail. I added unit test for SourceIdHandler in  here-naksha-lib-handlers and all other handlers which are not covered by te tests inside this module returned coverage 0.